ldm/invoke/config/invokeai_update.py

```diff
@@ -62,17 +65,26 @@ def main():
     welcome(versions)
 
     tag = None
-    choice = Prompt.ask('Choice:',choices=['1','2','3'],default='1')
+    branch = None
+    release = None
+    choice = Prompt.ask('Choice:',choices=['1','2','3','4'],default='1')
     
     if choice=='1':
-        tag = versions[0]['tag_name']
+        release = versions[0]['tag_name']
     elif choice=='2':
-        tag = 'main'
+        release = 'main'
     elif choice=='3':
-        tag = Prompt.ask('Enter an InvokeAI tag or branch name')
+        tag = Prompt.ask('Enter an InvokeAI tag name')
+    elif choice=='4':
+        branch = Prompt.ask('Enter an InvokeAI branch name')
 
     print(f':crossed_fingers: Upgrading to [yellow]{tag}[/yellow]')
```

Comment:
![image](https://user-images.githubusercontent.com/2433541/229249181-e8f1c148-43fe-4c05-9a34-f02ae98b09f8.png)


```suggestion
    print(f':crossed_fingers: Upgrading to [yellow]{tag if tag else release}[/yellow]')
```